cves/kernel/CVE-2014-4943.yml

```diff
@@ -19,26 +19,26 @@ curated_instructions: |
   This will enable additional editorial checks on this file to make sure you
```

Comment:
+3 upvotes CVE-2014-4943

Reply:
+3 from me as well, not enough people in my cohort.